src/containers/sidebars/FeaturesList.js

```diff
@@ -0,0 +1,67 @@
+import React, {Component} from "react";
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+import listOfModifiers from "../../constants/listOfModifiers";
+import listOfCollectors from "../../constants/listOfCollectors";
+import listOfDataFilters from "../../constants/listOfDataFilters";
+import listOfComparators from "../../constants/listOfComparators";
+import loadAndDisplayItems from "../../functions/loadAndDisplayItems";
+import toggleItemsVisibility from "../../functions/toggleItemsVisibility";
+
+class FeaturesList extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      componentClass: 'feature ' + this.props.type,
+      };
+  }
+
+  getListToLoad() {
+    switch(this.props.type.toLowerCase()) {
```

Comment:
I would suggest doing mapping of them somewhere and then use that mapping instead of switch

Reply:
Could you elaborate on that one?